### PR TITLE
feat(chart/ember): added probes

### PR DESCRIPTION
**Pull request by c0rydoras**

### CODE REVIEW

File: charts/outdated/templates/service.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ metadata:
     app.kubernetes.io/component: api
 spec:
   type: {{ .Values.api.service.type }}
+  publishNotReadyAddresses: true
```
Comment (hairmare) [inline charts/outdated/templates/service.yaml:10]: are you sure you need this? setting it to true will make the service send requests to the service that might fail

File: api/outdated/settings.py [hunk 2]
```diff
@@ -202,3 +203,13 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
 
 # Variables used only in testing
 VALIDATE_REMOTES = True
+
+# health checks
+WATCHMAN_CHECKS = env.list(
+    "WATCHMAN_CHECKS",
+    default=(
+        "watchman.checks.caches",
+        "watchman.checks.databases",
```
Comment (hairmare) [inline api/outdated/settings.py:212]: do you know what this does to check the database?

**Comment by hairmare** [comment]
one more watchman question... do you know if it would be possible to add a check that considers the availability of the configure oidc idp?

**Comment by hairmare** [comment]
oh, and also... the chart-testing phase is doing an awful lot of "Pulling image"... might that be an issue?

File: charts/outdated/templates/service.yaml (hunk 1 above)
Comment (c0rydoras) [reply charts/outdated/templates/service.yaml:10]: It shouldn't be necessary, without it i couldn't get it to run locally.

**Comment by hairmare** [comment]
ups

File: api/outdated/settings.py (hunk 2 above)
Comment (c0rydoras) [reply api/outdated/settings.py:212]:
```py
@check
def _check_database(database):
    connection = connections[database]
    with connection.cursor() as cursor:
        cursor.execute("SELECT 1")
    return {database: {"ok": True}}
```

File: charts/outdated/templates/service.yaml (hunk 1 above)
Comment (c0rydoras) [reply charts/outdated/templates/service.yaml]: its gone now and stuff works

**Comment by hairmare** [comment]
> watchman? 99-configure-oidc.sh in the frontend or do you mean something else?

it depends on the use-case. does the backend check jwt tokens via keycloak api (eg. by asking the userinfo endpoint).

if they backend can crash if keycloak isn't available, a checkout would be nice...

for the frontend it doesn't need many checks since it's just static page hosting anyway and the sh is fine.

**Comment by c0rydoras** [comment]
@hairmare i am in favour of an OIDC check but would like to do it in a seperate PR

**Comment by hairmare** [comment]
> @hairmare i am in favour of an OIDC check but would like to do it in a seperate PR

works for me!

File: api/outdated/settings.py (hunk 2 above)
Comment (hairmare) [reply api/outdated/settings.py:212]: nice, that's what i was expecting

**Review by hairmare — APPROVED** [review]

**Pull request merged**